src/Api/RefundPaymentApi.php

```diff
@@ -28,14 +28,14 @@ public function refund(
         string $token,
         string $paymentId,
         string $payPalAuthAssertion,
-        string $invoiceNumber,
+        string $invoiceId,
```

Comment:
I would be for not changing the argument name, as it could be a potential BC Break:
```suggestion
        string $invoiceNumber,
```